_rules/SC1-2-2-video-audio-alternative.md

```diff
@@ -0,0 +1,109 @@
+---
+name: Vídeo audio alternative
+description: |
+  This rule checks that video elements have an alternative for the auditive information.
+
+success_criterion:
+- 1.2.2
+
+atomic_rules:
+- SC1-2-video-media-alternative
+- SC1-2-2-video-caption
+
+authors:
+- Wilco fiers
+- Brian Bors
+
+---
+
+## Aggregation Definition
+
+### Applicability
+
+The rule applies to any [non-streaming][#non-streaming-video-element] `video` element that is [visible on the page](#visible-on-the-page), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [Video as a media alternative for text](#SC1-2-video-media-alternative)
+- [Video captions](#SC1-2-2-video-caption)
+
+## Assumptions
+
+This rule assumes that a mechanism is available to start the video and that the video element is not simply used to display the poster.
```

Comment:
I think "simply used to display the poster" is confusing. Maybe "This rule assumes that the video element is used to play a video (for example, not only used to display and image), and that there is a mechanism to start the video." instead?